src/neovim/neovim.ts

```diff
@@ -4,20 +4,23 @@ import { Position } from './../common/motion/position';
 import { Register, RegisterMode } from '../register/register';
 import { TextEditor } from '../textEditor';
 import { VimState } from './../state/vimState';
-import { attach, Nvim } from 'promised-neovim-client';
 import { configuration } from '../configuration/configuration';
 import { dirname } from 'path';
 import { exists } from 'fs';
 import { logger } from '../util/logger';
 import { spawn, ChildProcess } from 'child_process';
+import { attach, Neovim } from 'neovim';
 
-export class Neovim implements vscode.Disposable {
+export class NeovimWrapper implements vscode.Disposable {
```

Comment:
why change all of these names?

Reply:
To not mix it up with the exposed interface from the neovim npm package, and it is a wrapper

I understand that there are other ways of doing imports in neovim.ts but this seemed to be the simplest and clearest to me